src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,359 @@
+                taskCommand = "todo " + taskDescription;
+                addTodoTask(taskCommand);
+                break;
+            case "D":
+                taskCommand = "deadline " + taskDescription;
+                addDeadlineTask(taskCommand);
+                break;
+            case "E":
+                taskCommand = "event " + taskDescription;
+                addEventTask(taskCommand);
+                break;
+            }
+            if (isMarkedDone) {
+                markTaskAsDone("done " + taskCounter);
+            }
+        }
+    }
+
+    private static File getTaskFile() throws IOException {
+        File taskFile = new File(filePath + "/taskLists.txt");
+        if (taskFile.createNewFile()) {
+            System.out.println("A new file has been created at " + filePath);
+        } else {
+            System.out.println(taskFile + " accessed.");
+        }
+        System.out.print("\033[H\033[2J");
+        System.out.flush();
+        return taskFile;
+    }
+
+
+    private static void handleInputs() {
+        String input = getInput();
+        while (!input.equals("bye")) {
+            //Splits the input string by WhiteSpace into an array of strings.
+            String[] splittedInput = input.split(" ");
+
+            try {
+                switch (splittedInput[0].toLowerCase()) {
+                case "todo":
+                    addTodoTask(input);
+                    printNewTaskMsg();
+                    break;
+                case "deadline":
+                    addDeadlineTask(input);
+                    printNewTaskMsg();
+                    break;
+                case "event":
+                    addEventTask(input);
+                    printNewTaskMsg();
+                    break;
+                case "list":
+                    printListMessage();
+                    break;
+                case "done":
+                    markTaskAsDone(input);
+                    break;
+                case "delete":
+                    deleteTask(input);
+                    break;
+                case "bye":
+                    return;
+                default:
+                    throw new DukeException("☹ OOPS!!! I'm sorry, but I don't know what that means." + System.lineSeparator()
+                            + "\tPlease enter a valid input!" + System.lineSeparator()
+                            + "\ti.e. todo, deadline, event, list, done or bye.");
+                }
+
+                updateFile();
+
+            } catch (DukeException | IOException e) {
+                System.out.println(e.getMessage());
+            }
+
+            input = getInput();
+        }
+//
+    }
+
+    private static String getInput() {
+        String input;
+        Scanner scanner = new Scanner(System.in);
+        input = scanner.nextLine();
+        printLineSeparator();
+        return input;
+    }
+
+    private static void addTodoTask(String input) throws DukeException {
```

Comment:
While provide comments for single  line is useful, you could also consider giving single-line method descriptors.  See [javadocs](https://se-education.org/guides/conventions/java/basic.html#comments).